programs/merkle-tree/src/config_accounts/merkle_tree_authority.rs

```diff
@@ -87,6 +94,7 @@ pub struct UpdateMerkleTreeAuthorityConfig<'info> {
     pub authority: Signer<'info>,
 }
 
+#[cfg(not(feature = "atomic-transactions"))]
```

Comment:
Why is update lock duration behind the feature flag.
Shouldn't the TransactionMerkleTree be the same regardless how we update it?

Reply:
With Poseidon syscall, we don't need the lock and lock durations.

Reply:
true, you are right